pkgs/by-name/ru/rustdesk/package.nix

```diff
@@ -35,42 +35,46 @@
   copyDesktopItems,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "rustdesk";
-  version = "1.3.1";
+  version = "1.3.8";
 
   src = fetchFromGitHub {
     owner = "rustdesk";
     repo = "rustdesk";
-    rev = version;
-    hash = "sha256-PioaSdvgJ9oXC5DAbl+em7rxcGx1om9+sjCMdrvox90=";
+    tag = "${finalAttrs.version}";
+    hash = "sha256-m1bFljZL8vNaugepVs8u1EWNpDLtxgSSZqKGQmgrmsA=";
+    fetchSubmodules = true;
```

Comment:
```suggestion
    fetchSubmodules = true;
    hash = "sha256-m1bFljZL8vNaugepVs8u1EWNpDLtxgSSZqKGQmgrmsA=";
```

Reply:
Hi,

Can you explain me why this change ?

Best Regards

Reply:
unwritten rules that nixpkgs commiters like to see certain things in certain order (helps when viewing hundreds of packages daily)

Reply:
hashes are usually written at the end, after all options that influence the hash.